source/operations/external-iam/configure-keycloak-identity-management.rst

```diff
@@ -0,0 +1,267 @@
+
+      curl -d "client_id=minio" \
+           -d "client_secret=secretvalue" \
+           -d "grant_type=password" \
+           http://keycloak-url:port/admin/realms/REALM/protocol/openid-connect/token
+
+2. Use the value returned as the ``access_token`` to access the Admin API:
+
+   .. code-block:: shell
+      :class: copyable
+
+      curl -H "Authentication: Bearer ACCESS_TOKEN_VALUE" \
+           http://keycloak-url:port/admin/realms/REALM/users/UUID
+
+   Replace the UUID with the unique ID for the user which you want to retrieve.
+   The response should resemble the following:
+
+   .. code-block:: json
+      
+      {
+         "id": "954de141-781b-4eaf-81bf-bf3751cdc5f2",
+         "createdTimestamp": 1675866684976,
+         "username": "minio-user-1",
+         "enabled": true,
+         "totp": false,
+         "emailVerified": false,
+         "firstName": "",
+         "lastName": "",
+         "attributes": {
+            "policy": [
+               "readWrite"
+            ]
+         },
+         "disableableCredentialTypes": [],
+         "requiredActions": [],
+         "notBefore": 0,
+         "access": {
+            "manageGroupMembership": true,
+            "view": true,
+            "mapRoles": true,
+            "impersonate": true,
+            "manage": true
+         }
+      }
+
+   MinIO would revoke access for an authenticated user if the returned value has ``enabled: false``, or if the Admin API returned no matching user.
+
+If you do not enable this functionality, the earliest point in time that MinIO could disable access for a disabled or removed user is when the last retrieved authentication token expires.
+
+3) Enable Keycloak Admin Support on MinIO
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+MinIO supports multiple methods for configuring Keycloak Admin API Support:
+
+- Using a terminal/shell and the :mc-cmd:`mc admin idp openid` command
+- Using environment variables set prior to starting MinIO
+
+.. tab-set::
+
+   .. tab-item:: CLI
+
+      You can use the :mc-cmd:`mc admin idp openid update` command to modify the configuration settings for an existing Keycloak service.
+      You can alternatively include the following configuration settings when setting up Keycloak for the first time.
+      The command takes all supported :ref:`OpenID Configuration Settings <minio-open-id-config-settings>`:
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc admin idp openid update ALIAS KEYCLOAK_IDENTIFIER \
+            vendor="keycloak" \
+            keycloak_admin_url="https://keycloak-url:port/admin"
+            keycloak_realm="REALM"
+
+      - Replace ``KEYCLOAK_IDENTIFIER`` with the name of the configured Keycloak IDP.
+        You can use :mc-cmd:`mc admin idp openid list` to view all configured IDP configurations on the MinIO deployment
+        
+      - Specify the Keycloak admin URL to :mc-conf:`keycloak_admin_url <identity_openid.keycloak_admin_url>` configuration setting
```

Comment:
_Specify... to_ reads a bit oddly to me, although I'm not sure what might be better given I'm not familiar with the context.
(Here and also next item.)

Reply:
To clarify my comment, it's more about the `to` bit. "Specify _thing_ *to* _URL_."  I think I'd be more likely to write it as "specify _thing_ *for* _URL_."

Maybe that better describes what I was thinking? I'm not entirely certain what `keycloak_admin_url` is, so perhaps I'm the one misunderstanding.

Reply:
Maybe
> Specify the Keycload admin URL in the ...

I agree that the `to` phrasing catches me by surprise.